temel_bilgiler/kurulumlar/bilgisayar_temelleri.md

```diff
@@ -0,0 +1,44 @@
+### Giriş
+
+Söylemeye gerek yok, kodlamayı öğrenirken zamanınızın çoğunu bilgisayar başında geçireceksiniz. Bu nedenle, bilgisayarınızla nasıl çalışacağınızı anlamak çok önemlidir. Bu derste, sizi hızlı bir şekilde bilgisayar temelleri konusunda bilgilendirecek bir kaynak tanıtacağız.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
+
+* Bilgisayar yazılımı ve donanımı
+* Ekran görüntüleri nasıl alınır
+* Güçlü parolaların önemi
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+
+İyi niyet topluluğu vakfı (GCF) kaynakları, temel bilgileri öğrenmeniz için mükemmeldir. Vakfın eşlik eden videoları, çizimleri vardır ve okunması kolaydır. Aşağıdaki okumalar yeterli olacaktır, ancak GCF'nin diğer kaynaklarına da bakabilirsiniz.
```

Comment:
Capitalization is needed for institutions, organizations and such. We should not use comma before or after a conjunction.

fix: İyi Niyet Topluluğu Vakfı'nın (GCF) kaynakları, temel bilgilere aşinalık kazanmak için mükemmel kaynaklardır. Size yardımcı olacak videolar ve çizimler barındırmasının yanı sıra okunmaları kolaydır. Aşağıdaki okumalar yeterli olacaktır ancak GCF'nin diğer kaynaklarına da bakabilirsiniz.